addons/website_profile/models/res_users.py

```diff
@@ -53,8 +53,15 @@ def _send_profile_validation_email(self, **kwargs):
             }
             params.update(kwargs)
             token_url = self.get_base_url() + '/profile/validate_email?%s' % urls.url_encode(params)
-            activation_template.sudo().with_context(token_url=token_url).send_mail(
-                self.id, force_send=True, raise_exception=True)
+            activation_template.sudo().with_context(
```

Comment:
OOC, any idea why the sudo ? Maybe we browse user details in the template ?

Reply:
Maybe we could add a comment "at this point, user access on template should not restrict tempalte rendering", something like that ?